### PR TITLE
Feature/ab2d 292 automate shared components for sandbox

### PR DESCRIPTION
**Pull request by lhanekam**
Automate shared components for development and sandbox environments

### Summary

_Automate shared components for Sandbox_

- Create network diagram for the sandbox deployment
- Add parametrization of AWS account number to automation
- Configure AWS CLI for sandbox
- Create script to determine next available subnet CIDR block
- Break out shared scripts from environment level scripts
- Apply name tags to all AWS components with consideration of shared vs environment specific
- Create terraform environment for dev and sandbox environments
- Break out create vpc scripting for sbdemo environment
- Update destroy environment script to handle network destruction of multiple environments
- Break out delete vpc scripting for sbdemo environment
- Add parameterization of EC2 instance type to the automation
- Implement a shared RDS solution for deploying Dev and Sandbox environments
- Implement a shared EFS solution for deploying Dev and Sandbox environments
- Create generated docker-compose.yml for dev and sandbox with DB settings
- Automate process to push environment specific docker images to ECR
- Implement a shared S3 solution for deploying Dev and Sandbox environments
- Implement a shared networking solution for deploying Dev and Sandbox environments
- Modify the "destroy environment" script to destroy orphaned components in internal test environment
- Add parameterization of linux user to the automation

Implement CloudWatch infrastructure monitoring (automation will be enabled when Pager Duty integration is added)

- Automate EC2 metric alarms
- Automate ApplicationELB metric alarms
- Automate RDS metric alarms

### Checklist

- [X] Tests and linting pass

### Security
What secure items does this touch, how does this PR affect our security posture, etc?

_No security issues in this pull request._

### Additional JIRA Tickets (optional)

AB2D-292
AB2D-73

### CODE REVIEW

File: Deploy/documentation/Mac-Developer-SemanticBits.md [hunk 1]
```diff
@@ -745,6 +746,19 @@
    $ brew upgrade node
    ```
 
+## Install or update jq
+
+1. Install Packer using HomeBrew
```
Comment (dkrylovsb) [inline Deploy/documentation/Mac-Developer-SemanticBits.md]: @lhanekam You must have meant `jq`

File: Deploy/documentation/Mac-Developer-SemanticBits.md [hunk 2]
```diff
@@ -745,6 +746,19 @@
...
+
+   ```ShellSession
+   $ brew install jq
+   ```
+
+2. Verify the packer installation by checking the version of packer
```
Comment (dkrylovsb) [inline Deploy/documentation/Mac-Developer-SemanticBits.md]: Here, too

File: Deploy/documentation/Mac-Developer-SemanticBits.md (hunk 1 above)
Comment (lhanekam) [reply Deploy/documentation/Mac-Developer-SemanticBits.md]: Thanks. The risks of copy and paste. :-) I have fixed in my new branch.

File: Deploy/documentation/Mac-Developer-SemanticBits.md (hunk 2 above)
Comment (lhanekam) [reply Deploy/documentation/Mac-Developer-SemanticBits.md]: Fixed here too.

**Comment by lhanekam** [comment]
Committed the jq fix that Denis identified

**Pull request merged**